src/client/app/components/RouteComponent.jsx

```diff
@@ -12,6 +12,7 @@ import LoginContainer from '../containers/LoginContainer';
 import AdminComponent from './AdminComponent';
 import NotFoundComponent from './NotFoundComponent';
 import GroupContainer from '../containers/groups/GroupMainContainer';
+import EditGroupComponent from './groups/EditGroupComponent';
```

Comment:
Please remove this. This was removed in groupUI, but somehow not in your branch.